gamemode/modules/chat/sv_chat.lua

```diff
@@ -131,9 +131,9 @@ function GM:PlayerSay(ply, text, teamonly) -- We will make the old hooks run AFT
     local DoSayFunc
 
     for k, v in pairs(self.OldChatHooks) do
-        if type(v) ~= "function" then continue end
+        if not isfunction(v) then continue end
 
-        if type(k) == "Entity" or type(k) == "Player" then
+        if isentity(k) or type(k) == "Player" then
```

Comment:
The player check is useless now that you've changed it to isentity: players will already pass the first condition.

Reply:
Really, it is.